### DISCUSSION TITLE
Iperf using open thread

### DISCUSSION THREAD

**Discussion by jchellap** [openthread/openthread#9569] [General]
Further to [iperf commands - for throughput / traffic #9507](https://github.com/orgs/openthread/discussions/9507)

Further one query, When i use Leader as server and Router as child and client, i see UDP messages going through between client and server...with some throughput values.
However when i start iperf3 on child as server and Leader as client, it doesn't work. No data exchange.
Is it not to be used? Any explanation for not working will help me understand the right method of usage.

**Comment by jchellap** [comment]
Any insight on the behavior will help in understanding the correct method of usage.

**Comment by jwhui** [comment]
Consolidating discussion to https://github.com/orgs/openthread/discussions/9507
